### PR TITLE
Proposals/early compliance

### PR DESCRIPTION
**Pull request by JieXiong9119**
See proposal
Implementation in schema added

### CODE REVIEW

**Review by macintoshpie — APPROVED** [review]

**Review by corymosiman12 — APPROVED** [review]
👍

**Comment by nllong** [comment]
@yan130 -- can you take a look at this one and let us know what you think?

**Review by yan130 — APPROVED** [review]

**Comment by nllong** [comment]
@JieXiong9119 -- can you resolve the conflicts on this PR?

**Review by JieXiong9119 — COMMENTED** [review]
> @JieXiong9119 -- can you resolve the conflicts on this PR?

Looking good now. Could you check it out again?

File: proposals/2021/Add EarlyCompliance.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# Add EarlyCompliance
+
+## Overview
+
+This proposal is to add the `EarlyCompliance` element as a child of `ReportType` element, with boolean options.
```
[the remark this answers is not in the document]
Comment (nllong) [reply proposals/2021/Add EarlyCompliance.md:5]: I think the EarlyCompliance is about beating the deadline, right? Looks like the UDF field was "Early Compliance"...

[the remark this answers is not in the document]
Comment (JieXiong9119) [reply proposals/2021/Add EarlyCompliance.md:5]: That's right. I think it's good to go?

**Pull request merged**